### PR TITLE
[Content] Add jobs and missions about intra-Sol transportation

### PR DESCRIPTION
**Pull request by roadrunner56**
**Content (Artwork / Missions / Jobs)**

## Summary
It's mentioned in the Mars description that there is a significant amount of travel within Sol, as people move from Earth to Mars in search of employment. Transport from Mars to Earth is also common, as many of those workers are temporary. I made some new jobs and a culture conversation about transport within Sol to reflect this. Apologies if I've messed up the destination and source, I might need some help with that.

### CODE REVIEW

**Comment by MasterOfGrey** [comment]
I wonder, if we're going to lean into this, if we maybe shouldn't add some fleets to this system that don't have hyperdrives.

Fleets that can only spawn from planets, and travel to and land on planets. A direct buff to in-system transport.

File: data/human/near earth jobs.txt [hunk 1]
```diff
@@ -155,3 +155,53 @@ mission "From Earth Day celebration [1]"
 		payment
 		payment 1000
 		dialog `Your passengers thank you for returning them to <planet>, and you collect your payment of <payment>.`
+
+
+mission "Transport farmers to Mars"
+	job
+	repeat
+	description "This group of <bunks> factory workers are hoping to start a new on <planet> as farmers. They will pay you <payment> to take them there."
```
Comment (EjoThims) [inline data/human/near earth jobs.txt:163]:
```suggestion
	description "This group of <bunks> factory workers are hoping to start anew on <planet> as farmers. They will pay you <payment> to take them there."
```

File: data/human/near earth jobs.txt [hunk 2]
```diff
@@ -155,3 +155,53 @@ mission "From Earth Day celebration [1]"
...
+
+mission "Transport tourists to Luna"
+	job
+	repeat
+	description "This family of <bunks> would like to see the historic sights on Luna. They will pay you <payment> to take them there."
+	passengers 2 2 .8
+	to offer
+		random < 50
+	source "Earth"
+	destination "Mars"
```
Comment (EjoThims) [inline data/human/near earth jobs.txt:185]:
```suggestion
	destination "Luna"
```

**Review by EjoThims — APPROVED** [review]
Nice flavorful addition, imo.

Couple of small things though.

**Review by MasterOfGrey — APPROVED** [review]

File: data/human/culture conversations.txt [hunk 3]
```diff
@@ -582,3 +582,31 @@ mission "Scientists at the Pub after a Conference"
...
+
+mission "Sol-Bound Shuttle"
+	minor
+	source 
+		system "Sol"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`	Upon entering the sprawling spaceport, you notice that a group of factory workers are talking to a captain. As you get closer, you notice that the captain's flagship, a Shuttle that has seen better days, lacks a hyperdrive.`
```
Comment (W1zrad) [inline data/human/culture conversations.txt:595]:
```suggestion
			`	Upon entering the sprawling spaceport, you notice that a group of factory workers are talking to a captain. As you get closer, you see that the captain's flagship, a Shuttle that has seen better days, lacks a hyperdrive.`
```

**Review by W1zrad — APPROVED** [review]
These all look good. Enjoyed having a flavor mission for context! Any last-minute changes you want to make before I merge?

**Comment by roadrunner56** [comment]
> Any last-minute changes you want to make before I merge?
No, I think it's ready to be merged now. Thanks for the suggestions!

**Pull request merged**